bloom/commands/git/import_upstream.py

```diff
@@ -87,7 +87,7 @@ def version_check(version):
     info(fmt("The latest upstream tag in the release repository is '@!{0}@|'."
          .format(last_tag.replace('@', '@@'))))
     # Ensure the new version is greater than the last tag
-    if parse_version(version) < parse_version(last_tag_version):
+    if parse(version) < parse(last_tag_version):
```

Comment:
If either suggestion above is taken then this change would need to be reverted or modified.